src/components/Booking/RequestButton/index.js

```diff
@@ -4,11 +4,16 @@ import { PropTypes as PT } from 'prop-types';
 import { UNOBLUE } from '../../../styles/colors';
 
 const RequestButton = (props) => {
-  const { updateHoliday, booked, submitRequest, loading } = props;
+  const { updateHoliday, booked, submitRequest, loading, remainingHolidays, potentialHolidays, booking, availableDaysForNewBooking } = props;
+  const { wasHalfDay, duration } = booking;
+  const bookedAvailableDays = availableDaysForNewBooking - potentialHolidays + duration;
+  const unbookedAvailableDays = availableDaysForNewBooking - potentialHolidays;
```

Comment:
I'm not sure the terms used here are very clear - maybe add a comment or let's discuss some alternatives

Reply:
I have renamed this to availableDays and put it into a ternary.